src/main/java/com/chimbori/crux/articles/PostprocessHelpers.java

```diff
@@ -203,4 +208,13 @@ private static void removeDisallowedAttributes(Element node) {
     }
   }
 
+  private static boolean isImportant(Node node) {
+      boolean result=false;
```

Comment:
Can you please reformat this block using the style guide as specified in CONTRIBUTING.md? Thanks!

Reply:
I *think* got this right...